components/storybook-stories-image-snapshots.js

```diff
@@ -0,0 +1,48 @@
+// Sample stories for image snapshot testing
+// Components commented twice are not currently compatible with jest-image-snapshot
```

Comment:
Why not?

Reply:
Various reasons, but I mostly spelled this out in the Pull Request description. Many of our components measure things or create elements on the DOM.